src/main/java/org/example/service/TableMaker.java

```diff
@@ -0,0 +1,44 @@
+package org.example.service;
+
+import org.example.model.Developer;
+
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.util.List;
+
+public class TableMaker {
+
+    DeveloperService developerService = DeveloperService.getInstance();
```

Comment:
Is it ok, that this field is package-private?

Reply:
fix